dll/win32/advapi32/advapi32.h

```diff
@@ -49,6 +49,16 @@
 #define FN_PROGRESSA FN_PROGRESS
 #endif
 
+static const unsigned char CRYPT_LMhash_Magic[8] =
```

Comment:
Does this solve the problem?
Because now everyone that includes this header gets his own private copy of it.

Reply:
Right, but this allows us to have both PCH and unity builds working without resorting to ifdefs.